modules/integration/tests-common/admin-clients/src/main/java/org.wso2.greg.integration.common.clients/LoggingAdminClient.java

```diff
@@ -0,0 +1,70 @@
+/*
+*Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*WSO2 Inc. licenses this file to you under the Apache License,
+*Version 2.0 (the "License"); you may not use this file except
+*in compliance with the License.
+*You may obtain a copy of the License at
+*
+*http://www.apache.org/licenses/LICENSE-2.0
+*
+*Unless required by applicable law or agreed to in writing,
+*software distributed under the License is distributed on an
+*"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*KIND, either express or implied.  See the License for the
+*specific language governing permissions and limitations
+*under the License.
+*/
+
+package org.wso2.greg.integration.common.clients;
+
+import org.apache.axis2.AxisFault;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.logging.admin.stub.LoggingAdminException;
+import org.wso2.carbon.logging.admin.stub.LoggingAdminStub;
+
+import java.rmi.RemoteException;
+
+public class LoggingAdminClient {
```

Comment:
Hi Imesh,
Thanks for pointing this. I have added the comments in the same PR.
Thanks